styles/elements/export-dialog.styl

```
@@ -2,6 +2,7 @@
 
     .created-image
         width 100%
+        max-width: 310px;
```

```suggestion
        max-width: 310px
```

Seems that `;` aren't used at all in the `.styl` files.

---

removed all semicolons